src/views/UserProfileView.vue

```diff
@@ -690,7 +690,7 @@ and (max-device-width: 480px)
 and (-webkit-min-device-pixel-ratio: 2) {
   .profile__page {
     .profile__actions {
-      top: 3.3rem;
+      top: 3.1rem;
```

Comment:
This should be 3rem, because there will appear a gap

![image](https://user-images.githubusercontent.com/632282/78671522-15b2b300-78e8-11ea-96d2-b8b39c9f31e9.png)

If you want to have some "spacing" I suggest adding a top-border with a background color